MVP Module/Source/Feed Presentation/FeedPresenter.swift

```
@@ -32,6 +32,6 @@ final class FeedPresenter {
 	}
 	
 	func didFinishLoadingFeed(with error: Error) {
-		loadingView.display(FeedLoadingViewModel(isLoading: false))
+		loadingView.display(FeedLoadingViewModel(isLoading: false, errorMessage: Localized.Feed.loadError))
```

On `didStartLoadingFeed`, the presenter should tell the view to hide the error message.

This way, all presentation state changes are controlled by the presenter. It's much easier to manage state when all changes are centralized in a single component (instead of scattered around in many components such as the view, controller, and presenters).

So the presenter control when to show and when to hide the error message:

```swift
func didStartLoadingFeed() {
	loadingView.display(FeedLoadingViewModel(isLoading: true))
	errorView.display(FeedErrorViewModel(errorMessage: .none))
}
```

```swift
func didFinishLoadingFeed(with error: Error) {
	loadingView.display(FeedLoadingViewModel(isLoading: false))
	errorView.display(FeedErrorViewModel(errorMessage: Localized.Feed.loadError))
}
```

Notice how the views are different properties too - to decouple the concept of loading and error (see my other comment).